egs/aspire/s5/local/multi_condition/aspire_prep_rir_noise_list.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
I have already prepared around 9 RIR databases for use in our experiments. We have figure out which of these can be redistributed by us and prepare an upload.